client/src/components/ListOfOrganization/ListOfOrganization.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Created by ravi on 9/24/17.
+ */
+import styles from './ListOfOrganization.css'
+import React from 'react'
+
+function ListOfOrganization () {
+  return (
+    <form className={styles.form}>
```

Comment:
Please do not use a form here. The LOGIN and SIGN UP buttons can be part of story [#151007470](https://www.pivotaltracker.com/story/show/151007470) instead. I think just a `<section><h1></h1><ul></ul></section>` would be fine.

Reply:
Okay. I thought it needs to be here, since the wireframe had the buttons. I will remove them from this change.